src/referencePackages/buildReferencePackages.proj

```diff
@@ -1,24 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(RepoRoot)src/dir.props" />
+  <Import Project="$(RepoRoot)source-built/PackageVersions.props" />
 
   <PropertyGroup>
     <ReferenceAssemblySourcePath>$(RepoRoot)src/referencePackages/src/</ReferenceAssemblySourcePath>
     <TextOnlyProjectsSourcePath>$(RepoRoot)src/textOnlyPackages/</TextOnlyProjectsSourcePath>
+    <TargetingPacksPath>$(RepoRoot)src/targetingPackages/</TargetingPacksPath>
+    <TargetingPacksDLLPath>$(TargetingPacksPath)DLL/</TargetingPacksDLLPath>
     <ArtifactsBinDir>$(RepoRoot)artifacts/bin/</ArtifactsBinDir>
     <ArtifactsObjDir>$(RepoRoot)artifacts/obj/</ArtifactsObjDir>
     <ArtifactsLogDir>$(RepoRoot)artifacts/log/</ArtifactsLogDir>
-    <ArtifactsRefPkgDir>$(RepoRoot)/artifacts/reference-packages/</ArtifactsRefPkgDir>
+    <ArtifactsRefPkgDir>$(RepoRoot)artifacts/reference-packages/</ArtifactsRefPkgDir>
+    <ArtifactsTFMPackTemp>$(RepoRoot)artifacts/TFMPack/DLL/</ArtifactsTFMPackTemp>
+    <Artifacts472DLLLib>$(ArtifactsTFMPackTemp)microsoft.targetingpack.netframework.v4.7.2/1.0.0/lib/net472/</Artifacts472DLLLib>
+    <!--Ilasm hardcoded path. Needs to be changed with something in sb-ref-packages repo -->
+    <IlasmToolPathSB>$(RepoRoot)/source-built/coreclr-tools/</IlasmToolPathSB>
   </PropertyGroup>
 
   <Import Project="$(ReferenceAssemblySourcePath)ProjectsToBuild.props" />
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Sdk.IL" Version="3.0.0-preview9.19421.2" />
```

Comment:
Is this being used?

Reply:
I think it won't be used now. Good catch. I'll get rid of it 👍